backend/app/crud/crud_content.py

```
@@ -402,3 +407,94 @@ def delete_content_item_sync(session: Session, id: uuid.UUID) -> ContentItem | N
+
+
+def deactivate_content_share(db: Session, *, content_share: ContentShare) -> ContentShare:
+    """Deactivate a content share link."""
+    content_share.is_active = False
+    db.add(content_share)
+    db.commit()
+    db.refresh(content_share)
+    return content_share
+
+
+def delete_content_share(db: Session, *, id: uuid.UUID) -> ContentShare | None:
```

Pylint (3.3.7)

[warning] Redefining built-in 'id'

(W0622)